src/containers/SidechainHome/sidechainhome.scss

```diff
@@ -0,0 +1,126 @@
+@import '../shared/css/variables.scss';
+
+.app {
+  position: relative;
+
+  .sidechain-main-page {
+    position: relative;
+    min-height: 600px;
+    color: $white;
+
+    .logo-content {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      justify-content: center;
+      padding: auto;
+      margin: 97px auto 80px;
```

Comment:
nit - and probably this is just me, but i find this to be readable:

```css
margin: 10px 5px;
```

and this:

```css
margin: 10px 5px 10px 5px;
```

but not this
```css
margin: 10px 5px 10px;
```